src/views/collection-view/helpers/format-sidebar-sections.ts

```diff
@@ -38,7 +48,36 @@ export function formatSidebarCategorySections(
 		}
 	)
 
-	return filterEmptySections(sidebarSectionsByCategory)
+	/**
+	 * If we have category sections with content, use those.
+	 */
+	const nonEmptySections = filterEmptySections(sidebarSectionsByCategory)
+	if (nonEmptySections.length > 0) {
+		return nonEmptySections
+	}
+
+	/**
+	 * Otherwise, since there are no other sidebar sections, then
```

Comment:
HCP doesn't seem to have any collections tagged with `c.category` to match any of the `CollectionCategoryOption` options. So, I've added a "remainder section" that kicks in for cases like HCP where we'd otherwise show an empty sidebar.